pkg/registry/registry_antrea.csv

```diff
@@ -36,4 +36,8 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 134,reversePacketDeltaCountFromDestinationNode,unsigned64,,current,,,,,,,,56506,
 135,reverseOctetDeltaCountFromDestinationNode,unsigned64,,current,,,,,,,,56506,
 136,tcpState,string,,current,,,,,,,,56506,
-137,flowType,unsigned8,,current,,,,,,,,56506,
+137,flowType,unsigned8,,current,Type of flow based on location of source and destination. 1-IntraNode 2-InterNode 3-ToExternal 4-FromExternal,,,,,,,56506,
+138,tcpStatePrev,string,,current,,,,,,,,56506,
+139,tcpStatePrevList,string,,current,,,,,,,,56506,
+140,ingressNetworkPolicyRuleAction,string,,current,,,,,,,,56506,
```

Comment:
should we do uint8 instead of string? There are only limited action types.

Reply:
I saw in Antrea they are using string, but you are right there are only 3 types. Are you suggesting using uin8 will help reduce length of our flow record?

Reply:
I think this is still pending.